SwissTransferFeatures/TransferDetailsView/TransferDetailsView.swift

```diff
@@ -19,13 +19,35 @@
 import SwiftUI
 
 public struct TransferDetailsView: View {
-    public init() {}
+    private let title: String
+
+    public init(title: String) {
+        self.title = title
+    }
 
     public var body: some View {
-        Text("TransferDetailsView")
+        ScrollView {
+            VStack(spacing: 24) {
+                TransferDetailsHeaderView()
```

Comment:
As we are in a private framework, do we want to just call these views HeaderView(), MessageView(), ...